code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -36,6 +56,73 @@
 	else
 		return QDEL_HINT_LETMELIVE
 
+/obj/structure/colossus_rune //Sanguine sigil: Used to call forth the colossus
+	name = "sanguine sigil"
+	desc = "A sickening glyph drawn amongst the ashes in fresh blood."
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "landing"
```

Comment:
Don't think you should reuse the "a boss is attacking please move" rune with something you are supposed to click.

Also a rune laying in an empty area is a bit dull